### DISCUSSION TITLE
What Are the Optimal Settings for both Near & Far to Use All 16 Shortcuts?

### DISCUSSION THREAD

**Discussion by vcbrewers** [rxhanson/Charmstone-Community#53] [Q&A]
I've just purchased a license and am trying to use both Near & Far to take advantage of all 16 shortcuts but I am struggling with understanding the "safe area/"outer safe area" concepts. Ideally, I would like to be able to:

1. Select "Near" Shortcuts with the least trackpad distance possible.
2. Trigger "Far" Shortcuts with the least amount of trackpad distance possible while still having the option  to trigger "Near" Shortcuts first. 
3. Exit the popover without needing to select a shortcut first.

Appreciate any guidance!

**Comment by rxhanson** [comment]
Thanks for purchasing!

> 1. Select "Near" Shortcuts with the least trackpad distance possible.

Move the `Min Dist to Appear` to the left (all the way left means it will appear without any cursor movement).

> 2. Trigger "Far" Shortcuts with the least amount of trackpad distance possible while still having the option to trigger "Near" Shortcuts first.

Move the `Far Selection Distance` to the left. You might have to play around with this a little bit to get it right, depending on where the `Min Dist to Appear` is set.

> 3. Exit the popover without needing to select a shortcut first.

Move the `Outer Safe Area` to the left. All the way to the right means that moving the cursor far away from the initial point will not close the Charmstone view.


All of these correspond to number values that can be read via `defaults` Terminal commands. If you're using the App Store version it would be 
`defaults read com.knollsoft.Charmstone`
The non-app store version is 
`defaults read com.knollsoft.CharmstonePro`

The names of the values are a little different than what's in the UI, though.
